lua/packer/util.lua

```diff
@@ -31,7 +31,14 @@ util.nonempty_or = function(opt, alt)
   end
 end
 
-util.is_windows = jit.os == 'Windows'
+util.is_windows = (function()
```

Comment:
Nitpick: does this need to be a function? Is there a reason to not to do this check at the top level?

Reply:
> Is there a reason to not to do this check at the top level?
Sorry I couldn't understand what you ment by top lavel .
We could just use
`util.is_windows = package.config:sub(1,1) == '\\'`
Do you want me to change it to that ?